### ISSUE TITLE
Menu bar

### ISSUE THREAD

**Issue by jesse-wei** [COMP523TeamD/HeadCircumferenceTool#9]
When switching to circumference window, the menu bar doesn't actually change to the circumference window menu bar.

I added a menu bar element to main.ui but not circumference.ui, yet that GUI element showed up on circumference window. See commit [9a41430](https://github.com/COMP523TeamD/HeadCircumferenceTool/commit/9a414304a0258b20f3aab653beee03cad5e0e880).

Also, I am 100% certain that exporting as PNG in circumference window should export the contoured slice since that's what `self.image` currently is. However, it exports the slice without a contour.

Therefore, the menu bar of the circumference window must be the main window's menu bar.

This might be caused by how I set up the stacked widget in main, not in QtDesigner? Also should test on Windows.

**Comment by jesse-wei** [comment]
tl;dr in src/GUI/main.py, the two windows definitely have distinct menu bars and distinct items in their menu bars. The problem must be that switching to a different window does not render the menu bar? Will test on Windows.

```
From within the STACKED_WIDGET,
        MAIN_WINDOW's QMenuBar: <PyQt6.QtWidgets.QMenuBar object at 0x13433d5a0>
        Number of QAction children in MAIN_WINDOW's QMenuBar: 4
        MAIN_WINDOW's QMenuBar's QAction children: [<PyQt6.QtGui.QAction object at 0x13433fa30>, <PyQt6.QtGui.QAction object at 0x13433fac0>, <PyQt6.QtGui.QAction object at 0x13433df30>, <PyQt6.QtGui.QAction object at 0x13433fb50>]

        Just switched to CIRCUMFERENCE_WINDOW
        CIRCUMFERENCE_WINDOW's QMenuBar: <PyQt6.QtWidgets.QMenuBar object at 0x13433ed40>
        Number of QAction children in CIRCUMFERENCE_WINDOW's QMenuBar: 5
        CIRCUMFERENCE_WINDOW's QMenuBar's QAction children: [<PyQt6.QtGui.QAction object at 0x13433fbe0>, <PyQt6.QtGui.QAction object at 0x13433fc70>, <PyQt6.QtGui.QAction object at 0x13433f760>, <PyQt6.QtGui.QAction object at 0x13433fd00>, <PyQt6.QtGui.QAction object at 0x13433fd90>]
        No duplicated MenuBar QAction items
```

```py
    if settings.DEBUG:
        print(f"From within the STACKED_WIDGET,")
        curr_widget: QtWidgets.QWidget = STACKED_WIDGET.currentWidget()
        curr_widget_menubar = curr_widget.findChildren(QtWidgets.QMenuBar)[0]
        curr_widget_menubar_qactions = curr_widget_menubar.findChildren(QtGui.QAction)
        main_window_menubar_qactions = set(curr_widget_menubar_qactions)
        print(f"\tMAIN_WINDOW's QMenuBar: {curr_widget_menubar}")
        print(f"\tNumber of QAction children in MAIN_WINDOW's QMenuBar: {len(curr_widget_menubar_qactions)}")
        print(f"\tMAIN_WINDOW's QMenuBar's QAction children: {curr_widget_menubar_qactions}")

        STACKED_WIDGET.setCurrentWidget(CIRCUMFERENCE_WINDOW)
        print("\n\tJust switched to CIRCUMFERENCE_WINDOW")

        curr_widget: QtWidgets.QWidget = STACKED_WIDGET.currentWidget()
        curr_widget_menubar = curr_widget.findChildren(QtWidgets.QMenuBar)[0]
        curr_widget_menubar_qactions = curr_widget_menubar.findChildren(QtGui.QAction)
        print(f"\tCIRCUMFERENCE_WINDOW's QMenuBar: {curr_widget_menubar}")
        print(f"\tNumber of QAction children in CIRCUMFERENCE_WINDOW's QMenuBar: {len(curr_widget_menubar_qactions)}")
        print(f"\tCIRCUMFERENCE_WINDOW's QMenuBar's QAction children: {curr_widget_menubar_qactions}")

        for circumference_qaction in curr_widget_menubar_qactions:
            if circumference_qaction in main_window_menubar_qactions:
                print("\tDuplicated MenuBar QAction items") 
        else:
            print("\tNo duplicated MenuBar QAction items")
```

**Comment by jesse-wei** [comment]
Fixed by setting native menu bar to False in the `.ui` files.

The menu bar worked as expected on Windows but not macOS. It's a [macOS quirk](https://stackoverflow.com/questions/28559730/menu-entry-not-shown-on-mac-os)💀.

This doesn't affect Windows since Windows uses a non-native menu bar anyway. But it makes the macOS menu bar look like a Windows menu bar.

It's probably possible to work around this while maintaining a native menu bar on macOS. Some methods in src/GUI/main.py would need to change to functions that take `MainWindow` or `CircumferenceWindow` as an argument (frankly, this would be better design anyway since `render_curr_slice` and `export_slice_as_img` is duplicated code in both classes and should be functions).
